packages/react-vapor/src/components/checkbox/CheckboxSelectors.ts

```diff
@@ -8,10 +8,11 @@ const get = (state: IReactVaporState, {id}: {id: string}) => {
     return _.findWhere(state.checkboxes, {id});
 };
 
-const getIsSelected = createSelector(
-    get,
-    (checkbox: ICheckboxState): boolean => checkbox && checkbox.checked
-);
+const getIsSelected = () =>

```

Comment:
This is a breaking change. I think it was fine the way it was before because there is no real performance gain to make it a function that returns a selector instead of a straight up selector.

Have a read at [the reselect documentation](https://github.com/reduxjs/reselect#sharing-selectors-with-props-across-multiple-component-instances) for a good explanation of when to use selectors vs selectors creators